src/index.html

```diff
@@ -5,9 +5,55 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="main" data-qa="card">
+      <div class="card">
+        <img
+          src="./images/imac.jpeg" alt="Imac"
+          class="card__image"
+        >
+        <a href="#" class="card__name">
+          APPLE A1419 iMac 27" Retina
+          <br>
+          5K Monoblock (MNED2UA/A)
+        </a>
+      </div>
+      <p class="code">Product code: 195434</p>
+
+      <div class="stars">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="stars__rating">Reviews: 5</p>
+      </div>
+
+      <p class="price">
+        <span class="price__text">Price:</span>
+        <span class="price__value">$2,199</span>
+      </p>
```

Comment:
should be as an card element
```suggestion
      <p class="card__price">
        <span class="card__price_text">Price:</span>
        <span class="card__price_value">$2,199</span>
      </p>
```